services/github/github-hacktoberfest.service.js

```diff
@@ -113,12 +116,24 @@ module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Ser
     }
   }
 
-  static render({ suggestedIssueCount, contributionCount, daysLeft }) {
+  static render({
+    suggestedIssueCount,
+    contributionCount,
+    daysLeft,
+    daysToStart,
+    year,
+    hasStarted = true,
+  }) {
+    if (!hasStarted) {
+      return {
+        message: `${daysToStart} ${maybePluralize('day', daysToStart)} til kickoff!`,
```

Comment:
I wasn't really sure what do with the message here, definitely open to suggestions/alternatives